test/cmdlineTests/asm_json_export_yul_arbitrary_source_index/output

```diff
@@ -0,0 +1,38 @@
+
+======= <stdin> (EVM) =======
+{
+    ".code":
+    [
+        {
+            "begin": -1,
+            "end": -1,
+            "name": "PUSHSIZE",
+            "source": -1
+        },
+        {
+            "begin": -1,
+            "end": -1,
+            "name": "PUSH",
+            "source": -1,
+            "value": "0"
+        },
+        {
+            "begin": -1,
+            "end": -1,
+            "name": "SSTORE",
+            "source": -1
+        },
+        {
+            "begin": -1,
+            "end": -1,
+            "name": "STOP",
+            "source": -1
+        }
+    ],
+    "sourceList":
+    [
+        "unknown-source-0",
+        "unknown-source-1",
+        "L0.sol"
+    ]
```

Comment:
Oh, one more thing. We should have a test that this output is actually accepted by asm import.

Reply:
I added the test that receives this as input using `--import-asm-json` and should produce it again using `--asm-json`